src/tests/Helpers/GatewayHelperTest.php

```diff
@@ -0,0 +1,380 @@
+<?php
+/**
+ * Class GatewayHelperTest
+ *
+ * @covers \Alma\Woocommerce\Helpers\GatewayHelper
+ *
+ * @package Alma_Gateway_For_Woocommerce
+ */
+
+namespace Alma\Woocommerce\Tests\Helpers;
+
+
+use Alma\Woocommerce\AlmaSettings;
+use Alma\Woocommerce\Builders\Helpers\GatewayHelperBuilder;
+use Alma\Woocommerce\Exceptions\AlmaException;
+use Alma\Woocommerce\Factories\CartFactory;
+use Alma\Woocommerce\Factories\CoreFactory;
+use Alma\Woocommerce\Gateways\Standard\StandardGateway;
+use Alma\Woocommerce\Helpers\CartHelper;
+use Alma\Woocommerce\Helpers\ConstantsHelper;
+use Alma\Woocommerce\Helpers\GatewayHelper;
+use Alma\Woocommerce\Helpers\PHPHelper;
+use Alma\Woocommerce\Helpers\ProductHelper;
+use project\Controller\TodoController;
+use WP_UnitTestCase;
+
+/**
+ * @covers \Alma\Woocommerce\Helpers\GatewayHelper
+ */
+class GatewayHelperTest extends WP_UnitTestCase {
+
+	/**
+	 * @var GatewayHelper
+	 */
+	protected $gateway_helper;
+
+	/**
+	 * @var \Mockery\MockInterface|GatewayHelperBuilder
+
+	 */
+	protected $gateway_helper_builder;
+
+	/**
+	 * @var \Mockery\MockInterface|CoreFactory
+
+	 */
+	protected $core_factory;
+
+
+	/**
+	 * @var \Mockery\MockInterface|ProductHelper
+
+	 */
+	protected $product_helper;
+
+	/**
+	 * @var \Mockery\MockInterface|CartHelper
+
+	 */
+	protected $cart_helper;
+
+	/**
+	 * @var \Mockery\MockInterface|CartFactory
+
+	 */
+	protected $cart_factory;
+
+	/**
+	 * @var \Mockery\MockInterface|AlmaSettings
+
+	 */
+	protected $alma_settings;
+
+	/**
+	 * @var \Mockery\MockInterface|PHPHelper
+
+	 */
+	protected $php_helper;
+
+
+
+	public function set_up() {
+		$this->gateway_helper_builder = \Mockery::mock( GatewayHelperBuilder::class )->makePartial();
+		$this->gateway_helper = $this->gateway_helper_builder->get_instance();
+
+		$this->core_factory = \Mockery::mock( CoreFactory::class )->makePartial();
+		$this->product_helper = \Mockery::mock( ProductHelper::class )->makePartial();
+		$this->cart_helper = \Mockery::mock( CartHelper::class )->makePartial();
+		$this->cart_factory = \Mockery::mock( CartFactory::class )->makePartial();
+		$this->alma_settings = \Mockery::mock( AlmaSettings::class )->makePartial();
+		$this->php_helper = \Mockery::mock( PHPHelper::class )->makePartial();
+	}
+
+	public function tear_down() {
```

Comment:
You need to add Mockery::close() to work fine

Reply:
Done